internal/snmp/translate.go

```diff
@@ -163,7 +162,7 @@ func SnmpTranslateCall(oid string) (mibName string, oidNum string, oidText strin
 
 		out, err = gosmi.GetNode(node)
```

Comment:
```suggestion
		out, err = module.GetNode(node)
```
or 
```suggestion
		out, err = gosmi.GetNode(node, module)
```

Reply:
Both of these will fail on pasring other mibs. loading the module in fixes the panic and the library keeps track of it from there

Reply:
What do you mean with "other mibs"? Other than what?

Reply:
if you change it to module some of the test mobs than fail to parse. By keeping it gosmi and just node all mibs in the test parse properly and there is no more panic

Reply:
Okay, will have a look at those mibs. Shouldn’t be the case.

Reply:
I couldn't figure out how to change the mib so it passed when module was passed as well. I also know you hate changing of the test so I didn't want to remove that mib from the test. A catch 22.

Reply:
I think it's okay in this case as what is being tested is still there

Reply:
Awesome! Thank you for working with me on these snmp issues :)